gravitee-io/gravitee-access-management
# feat: invalidate token/session when username has changed

## :id: Reference related issue. 

https://gravitee.atlassian.net/browse/AM-384

## :pencil2: A description of the changes proposed in the pull request

This PR enables cancelling session/token when the user has its username changed

## :memo: Test scenarios 

- Login with a domain user via the Login flow
- Change their username
- re-initiate the login flow with the same user
- The user should be redirected to login

- Login with a organization user
- Change the username in organization settings
- The user should be redirected to login page


Still in Draft to add Jest tests for the scenarios
